x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/function/FunctionRegistry.java

```diff
@@ -421,4 +421,26 @@ public static FunctionDefinition def(Class<? extends Function> function, Functio
     protected interface CastFunctionBuilder<T> {
         T build(Source source, Expression expression, DataType dataType);
     }
+
+    @SuppressWarnings("overloads")  // These are ambiguous if you aren't using ctor references but we always do
+    public static <T extends Function> FunctionDefinition def(Class<T> function,
```

Comment:
How many arguments does wildcard expect?

Reply:
at least two, but it's unbounded in the maximum number
https://eql.readthedocs.io/en/latest/query-guide/functions.html#wildcard